cypress/support/fragments/data-export/selectJobProfile.js

```diff
@@ -26,7 +36,48 @@ export default {
     });
   },
 
-  verifySearchButton() {
-    cy.expect(searchButton.exists());
+  verifySearchButton(isDisabled) {
+    cy.expect(Button('Search', { disabled: isDisabled }).exists());
+  },
+
+  searchForAJobProfile(profile) {
+    cy.get(jobProfilescSearchId).type(profile);
+    cy.get(xIconSelector).should('exist');
+    cy.do([Button('Search', { disabled: false }).click()]);
```

Comment:
move all the Button('Search') to a const variable

Reply:
Done.